.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+sudo: false
+python:
+  - "3.3"
+addons:
+  postgresql: "9.3"
+before_script:
+  - psql -c 'create database feedthefox_db;' -U postgres
+install:
+  - pip install -r requirements.txt
```

Comment:
Also let's remove the ``docs`` entry from ``tox`` setup since we don't have any at the moment.